README.md

```diff
@@ -1,12 +1,65 @@
 # Commit Analyzer
 
-This is a simple tool, which takes the output of `git log
---output=outputfile.txt` and analyzes how much time somebody worked on it. E.g.
-simply call `commit-analyzer outputfile.txt --author-equals "wert007"` to see
-all commits, that fit the criteria and how much time there was put in it.
-
-You can also use the output of `git log --numstat --output=outputfile.txt` and
-if you specify an output parameter in commit analyzer (e.g. `commit-analyzer
-outputfile.txt --output "commits-and-loc-by-date.csv"`) it will generate an csv
-with the date, the number of commits made on that day and the loc changes that
-have been made.
\ No newline at end of file
+## Description
+
+This is a simple tool which takes the output of `git log` and analyzes how much
+time somebody worked on the project.
+
+For instance, simply call the following lines to see all commits that fit the
+criteria and how much time was spend creating them.
+
+```
+git log --output=outputfile.txt
+commit-analyzer outputfile.txt --author-equals "wert007"
+```
+
+You can also use the output of `git log --numstat` and give `commit-analyzer`
+an output file such that it will generate a CSV with the date, the number of
+commits made on that day and the loc changes that have been made.
+
+```
+git log --numstat --output=outputfile.txt
+commit-analyzer outputfile.txt --output "commits-and-loc-by-date.csv"
+```
+
+## License
+
+This project's license is **MIT**.  The license can be found `LICENSE` in the
+main directory of this repository.  Its text is as follows:
+
+> MIT License
+>
+> Copyright (c) 2022 Pablo Hamacher
+>
+> Permission is hereby granted, free of charge, to any person obtaining a copy
+> of this software and associated documentation files (the "Software"), to deal
+> in the Software without restriction, including without limitation the rights
+> to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+> copies of the Software, and to permit persons to whom the Software is
+> furnished to do so, subject to the following conditions:
+>
+> The above copyright notice and this permission notice shall be included in all
+> copies or substantial portions of the Software.
+>
+> THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+> IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+> FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+> AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+> LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+> OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+> SOFTWARE.
```

Comment:
Why are you adding the license again? If anyone is interested, they can check the `LICENSE` file anyway.

The `README.md` should in my opinion answer any question, that people might have checking out the repository. The `LICENSE` is not one of these imho.

Reply:
There are some projects which use a license that is either too unusual or too new that GitHub would be able to prompt the license's name on the project's welcome page.  That makes it difficult to use these projects.  Before using the work of anybody else, you ***always*** need to check if it is compatible with yours.  (This is also why I asked you to add a license since without one, I do not even know *whether* I am allowed to use your software, at all).

With such a section in your repository's README, every visitor will see directly what rights they have to share, modify and / or use your work.

Reply:
By the way:  this becomes even more critical when authoring an extension to VS Code-like IDEs, such as VS Code or VSCodium.  On the marketplace of Microsoft, it sometimes feels like ten minutes you need just to find out which license a plugin has.  Since the license also correlates with privacy policies and limitations of applications of the plugin in some circumstances, it is better to add at least a single word in the README somewhere such that your users do not need to do complete research about the terms of usage.

Reply:
But anyway, I can reduce it to a single line:

```
This project's license is **MIT**.
```

As most of the other points, this style originates from my experiences with GPL 2.0.  It is better to have such pieces of information, even if you literally add the license header to toilet paper, than omitting it in a single configuration file.

Reply:
All of the things you described to not apply here 😅 

But I am fine with that one liner. 😄

Reply:
Interesting.

But I think this project does fine with its `README.md` as its sole documentation. One thing, that could be improved might be in code documentation, but everything else is not needed at all I think. 😅

Reply:
I just removed the section.